src/sources.js

```diff
@@ -323,6 +325,16 @@ class S3Source extends Source {
     }
     return object.Body;
   }
+  parseOverviewMarkdown(overviewMarkdown) {
+    const frontMatter = yamlFront.loadFront(overviewMarkdown);
+    if (!frontMatter.title || !frontMatter.byline) {
+      throw new Error("Incorrectly formatted frontmatter in overview file");
```

Comment:
This error will never be displayed (the catch block following where `parseOverviewMarkdown` is called does nothing with the error object). Given that groups users won't have access to the server logs, in addition (or instead of) logging it server-side we should forward a helpful message as a value of the JSON response and then log this in the browser console.